reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedModelMapper.kt

```diff
@@ -445,13 +455,27 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
                 issues += addIssues(this.issues, EvaluatedOrtIssueType.ANALYZER, dependency, null, packagePath)
             }
 
+            val children = visitDependencies { dependencies ->
+                dependencies.map { node ->
+                    if (!deduplicateDependencyTree || node.getInternalId() !in visitedNodes) {
```

Comment:
How about introducing a variable for `node.getInternalId()`? (Currently, it get's called either once or twice.)